src/scripts/migrations/create-database.ts

```diff
@@ -0,0 +1,27 @@
+import { inversifyContainer } from '../../containers/inversify.container'
+import { LoggerService } from '../../services/logger.service'
+import { createRelationalDbClient } from '../../modules/relational-db/create-relational-db-client'
+
+const loggerService = inversifyContainer.get<LoggerService>(LoggerService)
+
+async function createDatabase(databaseName: string) {
+    const db = createRelationalDbClient('postgres')
+    try {
+        await db`CREATE DATABASE ${db(databaseName)}`
+    } catch (error: any) {
```

Comment:
had to add any here as i am not sure why it's giving me error on unknown which is the default type of error in try catch. Still haven't looked into it. If anyone knows what the issue is do let me know